app/webauthn/rest.py

```diff
@@ -0,0 +1,74 @@
+from flask import Blueprint, jsonify, request
+
+from app.dao.users_dao import get_user_by_id
+from app.dao.webauthn_credential_dao import (
+    dao_create_webauthn_credential,
+    dao_delete_webauthn_credential,
+    dao_get_webauthn_credential_by_id,
+    dao_update_webauthn_credential_name,
+)
+from app.errors import InvalidRequest, register_errors
+from app.schema_validation import validate
+from app.webauthn.webauthn_schema import (
+    post_create_webauthn_credential_schema,
+    post_update_webauthn_credential_schema,
+)
+
+webauthn_blueprint = Blueprint('webauthn', __name__, url_prefix='/user/<uuid:user_id>/webauthn')
+register_errors(webauthn_blueprint)
+
+
+@webauthn_blueprint.route('', methods=['GET'])
+def get_webauthn_credentials(user_id):
+    user = get_user_by_id(user_id)
+    return jsonify(data=[cred.serialize() for cred in user.webauthn_credentials]), 200
+
+
+@webauthn_blueprint.route('', methods=['POST'])
+def create_webauthn_credential(user_id):
+    data = request.get_json()
+    validate(data, post_create_webauthn_credential_schema)
+    webauthn_credential = dao_create_webauthn_credential(
+        user_id=user_id,
+        name=data['name'],
+        credential_data=data['credential_data'],
+        registration_response=data['registration_response']
+    )
+
+    return jsonify(data=webauthn_credential.serialize()), 201
+
+
+@webauthn_blueprint.route('/<uuid:webauthn_credential_id>', methods=['POST'])
+def update_webauthn_credential(user_id, webauthn_credential_id):
+    data = request.get_json()
+    validate(data, post_update_webauthn_credential_schema)
+
+    webauthn_credential = dao_get_webauthn_credential_by_id(webauthn_credential_id)
+
+    user = get_user_by_id(user_id)
+
+    check_credential_belongs_to_user(webauthn_credential.user_id, user.id)
```

Comment:
Minor: we could rework this to be part of `dao_get_webauthn_credential_by_id` instead, by making it scoped to the user. Then we'd correctly get a `404` if we tried to update or delete a credential for a different user, rather than a `400`.

Reply:
that's reasonable. we have precedence for doing this too: 

https://github.com/alphagov/notifications-api/blob/master/app/dao/jobs_dao.py#L52-L53